### ISSUE TITLE
Even more screen real estate wasted - please make this optional

### ISSUE THREAD

**Issue by kdekooter** [mastodon/mastodon-android#901]
After the increase of the line height (see #791) even more screen real estate is wasted by indenting the content of posts. 

For comparison a screenshot of old vs new:

![PHOTO-2024-10-05-11-20-19](https://github.com/user-attachments/assets/8904b729-1b01-48cb-a9a1-c4a4a11118f4)

**Comment by Offerel** [comment]
I'm pretty shocked at what that looks like now. I hope this can be reversed or at least made optionally. Especially if you have a lot of photography topics in the timeline, photos just look ugly.

I can't imagine something like that came to a designer's mind.

**Comment by antimundo** [comment]
I agree, the timeline should extend to its full width, why waste so much space?
- Images look way smaller this way.
- Text takes up a lot of vertical space because it has to be wrapped in a thinner area.
- Everything is oddly not centered, and slightly shifted to one side

**Comment by two-jays** [comment]
Please consider adding this Feature Back into the App!
I would Prüfer to use the app from the original devs but with that much unnecessary space wasted...
Please do an easy fix
